framework/set/setRKE2K3sConfig.go

```diff
@@ -124,20 +89,54 @@ func SetRKE2K3s(clusterName, k8sVersion, psact string, nodePools []config.Nodepo
 		machineConfigBlockBody := machineConfigBlock.Body()
 
 		kind := hclwrite.Tokens{
-			{Type: hclsyntax.TokenIdent, Bytes: []byte(`rancher2_machine_config_v2.rancher2_machine_config_v2.kind`)},
+			{Type: hclsyntax.TokenIdent, Bytes: []byte("rancher2_machine_config_v2.rancher2_machine_config_v2.kind")},
 		}
 
 		machineConfigBlockBody.SetAttributeRaw("kind", kind)
 
 		name := hclwrite.Tokens{
-			{Type: hclsyntax.TokenIdent, Bytes: []byte(`rancher2_machine_config_v2.rancher2_machine_config_v2.name`)},
+			{Type: hclsyntax.TokenIdent, Bytes: []byte("rancher2_machine_config_v2.rancher2_machine_config_v2.name")},
 		}
 
 		machineConfigBlockBody.SetAttributeRaw("name", name)
 
 		count++
 	}
 
+	upgradeStrategyBlock := rkeConfigBlockBody.AppendNewBlock("upgrade_strategy", nil)
+	upgradeStrategyBlockBody := upgradeStrategyBlock.Body()
+
+	upgradeStrategyBlockBody.SetAttributeValue("control_plane_concurrency", cty.StringVal(("10%")))
```

Comment:
same comment for defaulting these strings.

Reply:
These are just the default values set in Rancher. They aren't defined values to be dynamically set like in the prior comment, so I left it as is.